python/cudf/cudf/core/_base_index.py

```diff
@@ -850,7 +850,7 @@ def notna(self):
         """
         raise NotImplementedError
 
-    def to_pandas(self, nullable=False):
+    def to_pandas(self, nullable: bool = False):
```

Comment:
**suggestion:** I think we should make all of these arguments keyword-only
```
def to_pandas(self, *, nullable: bool = False):
  ...
```

Rationale: API safety in the case when one can pass more than just `nullable` as an argument. WDYT?

Reply:
Agreed. Does that mean I should label this PR as `breaking`?

Reply:
This is probably an API break, yes.